projects/desktop/src/components/atoms/common/Alert.tsx

```diff
@@ -6,9 +6,16 @@ import { getRawTheme } from '../../../redux/helpers/theme'
 import { IAlertColor } from '../../../interfaces/alerts'
 import { ITheme } from '../../../interfaces/theme'
 
+type PropsValue = string | undefined
+interface IAlertProps {
+    bgColor?: PropsValue;
+    color?: PropsValue;
+    borderColor?: PropsValue;
+}
+
 const AlertBox = styled.div<IAlertColor>`
-    background: ${props => props.bgColor};
-    color: ${props => props.color};
+    background: ${(props: IAlertProps): PropsValue => props.bgColor};
```

Comment:
Is it not better to put `props.bgColor ?? ''` and remove the `| undefined` of the `PropsValue` type?